engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigrationState.java

```diff
@@ -356,7 +356,8 @@ public void migrateElementInstancePopulateProcessInstanceByDefinitionKey() {
   @Override
   public void markMigrationFinished(final String identifier) {
     migrationIdentifier.wrapString(identifier);
-    migrationStateColumnFamily.insert(migrationIdentifier, new MigrationTaskState());
+    migrationStateColumnFamily.insert(
+        migrationIdentifier, new MigrationTaskState().setState(State.FINISHED));
```

Comment:
🔧 We can extract the FINISHED migration task state to a final field and reuse it